keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -591,6 +588,18 @@ service Provisioner {
 This call is meant to retrieve the unique provisioner Identity. This identity will have to be set in `BucketRequest.Provisioner` field in order to invoke this specific provisioner.
 
 ```
+message Protocol {
+	// ProtocolName is the name of the protocol
+    ProtocolName name = 1;
+	// version is the name of the protocol version
+    string version = 2;
+	oneof type {
+		S3Parameters s3 = 3;
```

Comment:
As above, is this "S3Parameters" to be used with *only* the S3 service, or is it about the S3 XML API (so maybe baseUrl would be here) and I can use it with GCS in S3-XML mode or my on-prem MinIO installation?

Reply:
It would be used by any s3 compatible service.